### DISCUSSION TITLE
원데이클래스에서 빈자리 발생 알림 앱

### DISCUSSION THREAD

**Discussion by ypd01018** [ondal-house/zu-story#13] [아이디어 선정 안건]
- 필라테스, 크로스핏, 1:1 PT 등에서 활용
- 노쇼시 빈자리 제공할 수 있는 플랫폼

Pros
- 위치 기반으로 한정하여 서비스 한다면 위치 기반 서비스 모델 적용도 가능할 것
- 계속해서 예약 현황을 유지해야 하기 때문에 ? 실시간 서비스 구현을 해볼 수 있을 것

Cons
- 구현 시에 더미 데이터를 이용하는게 아니라 실제 구상 서비스대로 한다면 업체와의 협의가 필요하단 점이 사실상 가장 어려울 것

**Comment by gychoi0916** [comment]
1) Mocking 해서 기능만 보여준다면 ? -> 개발 의욕과 방향성을 잡기가 쉽지 않을 // 오히려 붙일 수 있는 아이템도 많을 것
